Assets/Scripts/UI/Overlay/OverlayMap.cs

```diff
@@ -351,11 +353,17 @@ private void Update()
             elapsed = 0f;
         }
 
+        if(currentOverlay != "None" && currentLayer != WorldController.Instance.cameraController.CurrentLayer)
+        {
+            Bake();
+            currentLayer = WorldController.Instance.cameraController.CurrentLayer;
+            elapsed = 0f;
+        }
         // TODO: Prettify.
```

Comment:
Hem.. it pushed it in my own fork. Sorry no idea for now how to help
https://github.com/sboigelot/ProjectPorcupine/commit/480432fbda9c93cbf6825c1c0a7bb3d0c18b5403